tests/phpunit/elementor/app/import-export/test-usage.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\App\ImportExport;
+
+use Elementor\App\Modules\ImportExport\Usage;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+class Test_Usage extends Elementor_Test_Base {
+
+	public function setUp()
+	{
+		parent::setUp();
+
+		remove_all_filters('elementor/tracker/send_tracking_data_params');
+
+		( new Usage() )->set_usage_data();
+	}
+
+	public function test_get_usage_data__without_any_data()	{
+		// Act.
+		$usage = apply_filters('elementor/tracker/send_tracking_data_params', []);
+
+		// Assert
+		$import_export_usage = $usage['usages']['import_export'];
+
+		$this->assertEquals( [], $import_export_usage['revert'] );
+	}
+
+	public function test_get_usage_data__adds_usage_without_delete_existing() {
+		// Arrange.
+		update_option( 'elementor_revert_sessions', [
+			[
+				'kit_name' => 'Kit Name',
+				'source' => 'Source',
+				'revert_timestamp' => time(),
+				'import_timestamp' => time() - 100,
+			],
+		] );
+
+		$expected_usage_data = [
+			[
+				'kit_name' => 'Kit Name',
+				'source' => 'Source',
+				'revert_timestamp' => time(),
+				'total_time' => 100,
+			],
+		];
+
+		// Act.
+		$usage = apply_filters( 'elementor/tracker/send_tracking_data_params', [
+			'usages' => [],
+			'something-that-should-stay' => 123,
```

Comment:
maybe also add `something-that-should-stay` to the usages array, because that's the one you actually modify

Reply:
Done